Updated EXAMPLES

vedranmiletic opened the pull request:
- removed or replaced missing docs
- removed docs that switched to using other tools
- added some new docs

tk0miya reviewed (changes requested):
Thank you for your work. LGTM!! But it seems CI gets errored. Could you check it, please?

vedranmiletic committed:
Updated EXAMPLES

- removed or replaced missing docs
- removed docs that switched to using other tools
- added some new docs

vedranmiletic commented:
> 
> 
> Thank you for your work. LGTM!! But it seems CI gets errored. Could you check it, please?

Sorry, should have checked it better. Should be OK now.

tk0miya commented:
Merged. Thank you for your contribution!
